dev/com.ibm.ws.security.utility_fat_one/fat/src/com/ibm/ws/security/utility/test/SecurityUtilityCreateSSLCertificateTest.java

```diff
@@ -0,0 +1,816 @@
+        assertTrue("Server keystore should exist", keystoreFile.exists());
+
+        runserver(commandOutput, sslTestServer, "");
+    }
+    
+    /**
+     * Tests creating an SSL certificate for a client and verifies it works.
+     */
+    @Test
+    public void testCreateCertificateForClient() throws Exception {
+        ProgramOutput commandOutput = runCreateSSLCert(
+            "--client=" + SSL_TEST_CLIENT_NAME, 
+            "--password=Liberty"
+        );
+        int returnCode = commandOutput.getReturnCode();
+        assertTrue(returnCode == SUCCESS_RC);
+
+        File keystoreFile = new File(libertyInstallRoot + "/usr/clients/" + SSL_TEST_CLIENT_NAME + "/" + KEYSTORE_PATH);
+        assertTrue("Client keystore should exist", keystoreFile.exists());
+
+        runclient(commandOutput, testClient);
+    }
+
+    /**
+     * Tests creating an SSL certificate with JKS keystore type.
+     */
+    @Test
+    public void testCreateWithKeyTypeJKS() throws Exception {
+        File jksFile = new File(libertyInstallRoot + "/usr/servers/" + SSL_TEST_SERVER_NAME + "/" + KEYSTORE_PATH_JKS);
+        if (jksFile.exists()) {
+            assertTrue("Failed to delete existing key.jks", jksFile.delete());
+        }
+        
+        ProgramOutput commandOutput = runCreateSSLCert(
+            "--server=" + SSL_TEST_SERVER_NAME,
+            "--password=Liberty",
+            "--keyType=JKS"
+        );
+        assertEquals(SUCCESS_RC, commandOutput.getReturnCode());
+        
+        runserver(commandOutput, sslTestServer, "");
+    }
+
+    /**
+     * Tests creating an SSL certificate with AES password encoding and custom key.
+     */
+    @Test
+    public void testCreateWithPasswordEncodingAESV1() throws Exception {
+        ProgramOutput commandOutput = runCreateSSLCert(
+            "--server=" + SSL_TEST_SERVER_NAME,
+            "--password=Liberty",
+            "--passwordEncoding=aes",
+            "--passwordKey=MySecretKey123"
+        );
+        assertEquals(SUCCESS_RC, commandOutput.getReturnCode());
+        Log.info(thisClass, testName.getMethodName(), "stderr:\n" + commandOutput.getStderr());
+        Log.info(thisClass, testName.getMethodName(), "stdout:\n" + commandOutput.getStdout());
+        Log.info(thisClass, testName.getMethodName(), "Return code: " + commandOutput.getReturnCode());
+
+        String aesEncryptionKey = "<variable name=\"wlp.password.encryption.key\" value=\"MySecretKey123\" />";
+
+        runserver(commandOutput, sslTestServer, aesEncryptionKey);
+
+
+    }
+
+    /**
+     * Tests creating an SSL certificate with AES V2 password encoding and a custom key.
+     */
+    @Test
+    public void testCreateWithPasswordEncodingAESV2() throws Exception {
+
+        byte[] keyBytes = new byte[32];
+        new SecureRandom().nextBytes(keyBytes);
+        String base64Key = Base64.getEncoder().encodeToString(keyBytes);
```

Comment:
lets delete 550-552 and use a string like "key123" instead for `--key`. `--key` is just a passphrase for derivation so i dont want to cause any confusion by generating a base64 key for it (even though it's valid input)